standalone/RL_plus_script.py

```diff
@@ -104,13 +117,18 @@ def action(self, action):
 
 
 def train():
-    env = gym.make(TRAIN_ENV)
-    env = PovOnlyObservation(env)
-    env = ActionShaping(env, always_attack=True)
-
+    if os.environ.get("WANDB_PROJECT"):
```

Comment:
Question because of my lack of wandb knowledge, but: is this a commonly used env variable for using wandb? I would stick with what official tutorials/docs recommend is best course of action (but I assume you already are doing it ^^)

Reply:
Modified to better match official documentation.